### PR TITLE
Use recommended numpy pinning recipe

### PR DESCRIPTION
**Pull request by Anthchirp**
Currently you require numpy 1.16 on the host to build, which means this
package will not be available on platforms that do not have numpy 1.16,
which includes Python 3.9.

This is following the recipe in
https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Anthchirp** [comment]
@conda-forge-admin, please rerender

**Comment by isuruf** [comment]
They are vendoring numpy headers, so this is not the solution.

**Comment by Anthchirp** [comment]
Thanks for checking!
Unfortunately I don't know what that means in terms of how to proceed from here.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,12 +19,12 @@ requirements:
     - python
     - pip
     - cython
+    - numpy >=1.16
```
Comment (ocefpaf) [inline recipe/meta.yaml:22]: You can remove the `>-1.16` and let our pinning use the min supported version, which is higher than 1.16. Having it there will end up building with the latest and only running with the latest.

Comment (Anthchirp) [reply recipe/meta.yaml]: okay, done

**Comment by ocefpaf** [comment]
> They are vendoring numpy headers, so this is not the solution.

:-(

Also not sure on how to proceed. Should we just remove the numpy build dependency then @isuruf?

**Comment by adrianeboyd** [comment]
We are no longer vendoring the numpy headers as of v0.7.3 and python 3.9 packages are now available.

**Pull request closed without merging**